diesel_codegen/src/lib.rs

```diff
@@ -23,12 +23,12 @@ pub fn register(reg: &mut rustc_plugin::Registry) {
         MultiDecorator(Box::new(identifiable::expand_derive_identifiable))
     );
     reg.register_syntax_extension(
-        intern("derive_Queryable"),
-        MultiDecorator(Box::new(queryable::expand_derive_queryable))
+        intern("derive_Insertable"),
+        MultiDecorator(Box::new(insertable::expand_derive_insertable))
     );
     reg.register_syntax_extension(
-        intern("insertable_into"),
-        MultiDecorator(Box::new(insertable::expand_insert))
+        intern("derive_Queryable"),
+        MultiDecorator(Box::new(queryable::expand_derive_queryable))
```

Comment:
Is there a reason you switched the order of the register calls (insert before query)? Same in `diesel_codegen_syntex/src/lib.rs`. It obviously works but I couldn't find a reason if the order makes a difference.